mongo/integration/mtest/mongotest.go

```diff
@@ -405,14 +405,16 @@ func (t *T) extractDatabase(coll Collection) *mongo.Database {
 
 // CreateCollection creates a new collection with the given configuration. The collection will be dropped after the test
 // finishes running. If createOnServer is true, the function ensures that the collection has been created server-side
-// by running the create command. The create command will appear in command monitoring channels.
+// by running the create command. The create command will not appear in command monitoring channels.
```

Comment:
All changes in this file are to ensure that the global client is used for administrative things like creating collections and failpoints. This is to make sure that bugs like this are not hidden. The existing test in `sessions_test.go` should have failed, but didn't because the `create` command being executed in the testing library got a valid cluster time from the server.